### ISSUE TITLE
Inconsistent behavior with nth() between Sass and libsass.

### ISSUE THREAD

**Issue by crswll** [sass/libsass#843]
The `nth` function in libsass doesn't seem to behave the same way as it does in Sass. In the example below `nth($list, 1)` results in `1/2` in libsass and `.5` in Sass. The `type-of()` on both is `number`.

Here's my demo code:

````scss

$list: 1/2 3 3*3 1+2;
$single: 1/2;

/*
  First Item In $list: #{nth($list, 1)};
  $single: #{$single};
  Second Item In $list: #{nth($list, 2)};
  Third Item In $list: #{nth($list, 3)};
  Fourth Item In $list: #{nth($list, 4)}; 
*/
  
````

Libsass output:

````
/*
  First Item In $list: 1/2;
  $single: 0.5;
  Second Item In $list: 3;
  Third Item In $list: 9;
  Fourth Item In $list: 3; 
*/
````

Sass output:

````
/*
  First Item In $list: 0.5;
  $single: 0.5;
  Second Item In $list: 3;
  Third Item In $list: 9;
  Fourth Item In $list: 3; 
*/

````
Here's the code on SassMeister if anyone feels like playing with it: http://sassmeister.com/gist/dc1053c2916736dafbb3

I had no idea how to search for this issue to make sure it didn't already exist.

**Comment by HugoGiraudel** [comment]
That's an odd one.

**Comment by xzyfer** [comment]
This is a known issue. It's fixed on master and will be 3.2.0.
